CONTRIBUTING.md

```diff
@@ -1,3 +1,14 @@
+# Good Development Resources
+
+- [Introduction to UE4 Plugins](https://wiki.unrealengine.com/An_Introduction_to_UE4_Plugins)
+- [MacOS Setup Guide](https://github.com/botman99/ue4-xcode-vscode-mac)
+- If you are new to programming in UE4, please see the Epic's [Programming Guide](https://docs.unrealengine.com/latest/INT/Programming/index.html)! 
```

Comment:
nit: i have a preference against the second person tense in documentations. Opinion?

Reply:
prefer imperative statements without a subject (you, the developer, they) in mind